opentracing-cdi/pom.xml

```diff
@@ -22,8 +22,21 @@
       <artifactId>opentracing-api</artifactId>
     </dependency>
     <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-tracerresolver</artifactId>
+    </dependency>
+    <dependency>
       <groupId>io.opentracing</groupId>
       <artifactId>opentracing-util</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-ejb</artifactId>
```

Comment:
The base interceptor is there. I'm thinking about extracting the interceptor into a `java-interceptor`, and making CDI and EJB use it, but I'm not convinced yet. The EJB dependency brings nothing with it, except the name "EJB", so, as of now, I'll keep it like this.